### ISSUE TITLE
need latest google play service version on emulator

### ISSUE THREAD

**Issue by mehrdad-abdoli** [butomo1989/docker-android#55]
Hi
How can i install and update google play service on emulator ?

**Comment by zoutepopcorn** [comment]
Yeah, I'm also interested.

I tried to boot it into recovery mode, but with ADB it fails.

**Comment by zoutepopcorn** [comment]
Update:

The Google Play Store is now officially preinstalled the Android Emulator. Make sure you are running the latest version of Android Studio 2.4. In the Android Studio AVD Manager choose a virtual device configuration that has the Google Play store icon next to it, and then select one of the system images that have the label "Google Play". See this release note: https://androidstudio.googleblog.com/2017/04/android-studio-24-preview-4-is-now.html

**Comment by butomo1989** [comment]
Hi @mehrdad-abdoli and @zoutepopcorn , 

Thank you for opening this issue. Why do you need google play / what kind of test or use case do you have? because I think you can install the app through .apk file using adb install command.

**Comment by mehrdad-abdoli** [comment]
google play service is need to be installed so some feautures in our app working

**Comment by mehrdad-abdoli** [comment]
@zoutepopcorn  I need play service in this repository android dockers

**Comment by greenerchen** [comment]
@mehrdad-abdoli @butomo1989 Hi, I was looking for how to install play store in the emulator as well. Here is what I found. When you list all available system images by sdkmanager, you may find the system images with play store which are same as what you find in the Android Studio AVD Manager.
```
sdkmanager --list --verbose
```

Available system images with play store:
system-images;android-24;google_apis_playstore;x86
system-images;android-25;google_apis_playstore;x86
system-images;android-26;google_apis_playstore;x86
system-images;android-27;google_apis_playstore;x86

You will got the latest google play store installed in the emulator after Installing the one with latest API and creating an avd with it either in the docker container or in Dockerfile before building a docker image.
```
echo y | sdkmanager "system-images;android-27;google_apis_playstore;x86"
echo n | avdmanager create avd \
    -k "system-images;android-27;google_apis_playstore;x86" \
    -n "emulator-27-playstore" -b x86 -g google_apis_playstore
```

**Comment by butomo1989** [comment]
Hi @greenerchen , 

I just checked it but it supports only for api 24 / android 7.0 and higher. :( so I need to put the logics to separate other android version that I support like android 5.0.1. It would be great if somebody can help me with it.

**Comment by greenerchen** [comment]
Hi @butomo1989 

Yea, it supports only for api 24 / android 7.0 and higher version on x86 processor. For arm process or prior versions, it might need manual play store installation like [this solution](https://infosectrek.wordpress.com/2017/01/17/installing-the-google-play-store-app-apk-on-the-android-emulator/) which i successfully used with the docker image butomo1989/docker-android-arm-7.0. Can you elaborate more about where to put the logics to separate other android versions for system images with play store?

**Comment by butomo1989** [comment]
Could somebody use this project: https://github.com/yeriomin/YalpStore and give a feedback? It would be great if you update the readme as well.

**Comment by butomo1989** [comment]
Fixed in [release 1.1](https://github.com/butomo1989/docker-android/releases/tag/1.1)
